docker-compose.yml

```diff
@@ -54,7 +54,14 @@ services:
     image: amazon/aws-cli
     env_file: .env
     command: ["--endpoint-url", "$S3_ENDPOINT", "s3api", "create-bucket", "--bucket", "$S3_BUCKET"]
-
+  clamav-server:
+    image: mkodockx/docker-clamav:alpine
+  clamav-rest:
+    image: lokori/clamav-rest
+    environment:
+      CLAMD_HOST: clamav-server
+    ports:
+      - "8081:8080"
```

Comment:
Should the entry in `.env.example` be port 8081 for docker then?

Reply:
No, 8081 is exposed to the host, but it is 8080 on the docker network. It couldn't be 8080 on the host because that is used by the backend. So other containers can still reach it at 8080. You can exec into the backend container and run `curl -F "name=test" -F "file=GranteeTTAPlan.csv" clamav-rest:8080/scan` and it returns fine.